src/components/Header.tsx

```diff
@@ -8,8 +8,12 @@ export interface HeaderProps {
   logOut?: string;
 }
 
+declare const require: any;
```

Comment:
I don't see where `require` is used.

Reply:
Line 17.

Reply:
Ah, right. I wonder if we still need that...